popup.html

```diff
@@ -201,8 +124,7 @@
      <p>
      <b>3.</b> Copy this HTML and paste into your webpage.
      </p>
-    <textarea id="fontHtml" style="width:100%; height: 70px;">
-    </textarea>
+    <textarea id="fontHtml"></textarea>
    </div>
 
    <div id="step4">
```

Comment:
Removing the reset link breaks the event listener. Are you removing this permanently or just until the reset feature can be fixed (ie. Should I comment out the event listener code for now or are we removing all the code for this feature)

Reply:
I'm fine with bringing the feature back if it feasible to get it working, but until it works, lets leave out of the UI. Comment out for now, since it seems like wed be close to getting v3 out if that wasnt a concern?

Reply:
Sounds good, and yes, it should be about ready.